src/composables/useFormatDateTime.js

```diff
@@ -58,14 +58,14 @@ export function useFormatDateTime(timestamp = Date.now(), opts = {}) {
 	const date = computed(() => new Date(unref(timestamp)))
 
 	const formattedFullTime = computed(() => {
-		const formatter = new Intl.DateTimeFormat(getCanonicalLocale(), wrappedOptions.value.format)
+		const formatter = new Intl.DateTimeFormat(getLanguage(), wrappedOptions.value.format)
```

Comment:
This does not work, for example I have configured Nextcloud to use English (US) lanuage, but German locale (I want 24h system and dd.mm.yyyy days but English texts).
Now I receive 12h times and English date format.

Reply:
So at least the full formatted time needs to use the users **locale** not language